HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5606,6 +5835,486 @@ def self.get_ceiling_fan_months(weather)
+      end
+
+      hpxml_bldg.heat_pumps.each do |heat_pump|
+        next if !system_ids.include?(heat_pump.id)
+        next if heat_pump.fraction_cool_load_served == 0
+
+        if heat_pump.fraction_heat_load_served == 0
+          breaker_spaces += get_breaker_spaces_from_power_watts_and_voltage(HVAC.get_dx_cooling_coil_power_watts_from_capacity(UnitConversions.convert(heat_pump.cooling_capacity, 'btu/hr', 'kbtu/hr')), voltage) # ODU; the ~2 we missed adding to heating
+        end
+      end
+
+    elsif type == HPXML::ElectricPanelLoadTypeWaterHeater
+      hpxml_bldg.water_heating_systems.each do |water_heating_system|
+        next if !system_ids.include?(water_heating_system.id)
+        next if water_heating_system.fuel_type != HPXML::FuelTypeElectricity
+        next if water_heating_system.is_shared_system
+
+        if water_heating_system.water_heater_type == HPXML::WaterHeaterTypeStorage
+          breaker_spaces += get_breaker_spaces_from_power_watts_and_voltage(watts, voltage)
+        elsif water_heating_system.water_heater_type == HPXML::WaterHeaterTypeHeatPump
+          breaker_spaces += get_breaker_spaces_from_power_watts_and_voltage(watts, voltage)
+        elsif water_heating_system.water_heater_type == HPXML::WaterHeaterTypeTankless
+          if hpxml_bldg.building_construction.number_of_bathrooms == 1
+            breaker_spaces += default_panels_csv_data['wh_tankless1'][voltage]['BreakerSpaces']
+          elsif hpxml_bldg.building_construction.number_of_bathrooms == 2
+            breaker_spaces += default_panels_csv_data['wh_tankless2'][voltage]['BreakerSpaces']
+          else # 3+
+            breaker_spaces += default_panels_csv_data['wh_tankless3'][voltage]['BreakerSpaces']
+          end
+        end
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeClothesDryer
+      hpxml_bldg.clothes_dryers.each do |clothes_dryer|
+        next if !system_ids.include?(clothes_dryer.id)
+        next if clothes_dryer.fuel_type != HPXML::FuelTypeElectricity
+
+        if clothes_dryer.is_vented
+          breaker_spaces += default_panels_csv_data['dryer'][voltage]['BreakerSpaces']
+        else # HP
+          breaker_spaces += default_panels_csv_data['dryer_hp'][voltage]['BreakerSpaces']
+        end
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeDishwasher
+      hpxml_bldg.dishwashers.each do |dishwasher|
+        next if !system_ids.include?(dishwasher.id)
+
+        if voltage == HPXML::ElectricPanelVoltage240
+          runner.registerWarning("PanelLoad '#{type}' has PanelLoad/Voltage (#{voltage}) not specified in default_panels.csv; PanelLoad/BreakerSpaces will be recalculated.")
+          default_panels_csv_data['dishwasher'][voltage] = {}
+          default_panels_csv_data['dishwasher'][voltage]['BreakerSpaces'] = get_breaker_spaces_from_power_watts_and_voltage(watts, voltage)
+        end
+        breaker_spaces += default_panels_csv_data['dishwasher'][voltage]['BreakerSpaces']
+      end
```

Comment:
Similar question down here. We've now assumed the power rating of a 120V dishwasher. Breaker spaces was not specified, so we enter this block. We should now calculate breaker spaces using our method with the 120V power rating and 240V specification, and throw a warning?